pasta_eln/GUI/table.py

```diff
@@ -269,10 +269,11 @@ def execute(self, command:list[Any]) -> None:
           if ret==QMessageBox.StandardButton.Yes:
             doc = self.comm.backend.db.getDoc(docID)
             for branch in doc['-branch']:
-              oldPath = self.comm.backend.basePath/branch['path']
-              if oldPath.exists():
-                newPath = oldPath.parent / f'trash_{oldPath.name}'
-                oldPath.rename(newPath)
+              if branch['path'] is not None:
+                oldPath = self.comm.backend.basePath/branch['path']
+                if oldPath.exists():
+                  newPath = oldPath.parent / f'trash_{oldPath.name}'
+                  oldPath.rename(newPath)
```

Comment:
So we do not remove the file, instead just give a trash_ suffix to it? Why do we keep this copy and are they removed at some point?

Reply:
Data removal is dangerous and hence, we never really do it, to prevent unintended actions.
First a data is moved to "trash_". If then the next item with exactly the same name gets removed, aka moved to "trash_' then the orginal data is really deleted: there is no trash_trash_...
This should be enough safeguard against removal: you might not want to remove content within a folder but if the folder is gone, normally the content would also be gone.
What I said applies to content on harddisk only. Content in the database is removed at first choice.